conda/pyhit/meta.yaml

```diff
@@ -1,4 +1,4 @@
-{% set version = "2024.05.08" %}
+{% set version = "2025.01.07" %}
```

Comment:
FYI, Changing anything in conda/pyhit is no longer necessary, as PyHIT is now being built as an output package in conda/wasp. Changes in this recipe will actually be ignored by Civet.

Ideally we should remove conda/pyhit. I would support and help in its removal if you would like to make that a part of this PR?

```bash
git rm -rf conda/pyhit
```

Reply:
We can also ignore my request, and I will remove it in another PR another day :)

Reply:
No, this makes sense and explains my confusion above with conda/pyhit and https://github.com/milljm/moose-pyhit.

Reply:
I can imagine! Plus having it be my own personal repository... gross.

My apologies not removing this when I moved PyHIT into the WASP Conda package.